docs/lab/01.md

```diff
@@ -2,4 +2,240 @@
+
+$$
+[I]_{SI} = A(Ampere)
+$$
+
+### Ohm's law
+
+The intensity(I) of the electric current passing through a resistor is directly proportional to the voltage(U) applied to the resistor and inversely proportional to its resistance value(R).
+
+<div align="center">
+![Ohm's Law](images/ohm.png)
+</div>
+
+$$
+I = \frac{U}{R};
+$$
+
+I = intensity of electric current(A)   
+U = electrical voltage(V)   
+R = circuit's resistance(&ohm;)
+
+### Kirchhoff's First Law
+
+The sum of currents flowing into a junction is always equal to the sum of currents flowing out of the same junction.
+
+$$
+\sum_{k=1}^{n} I_k = 0
+$$
+
+<div align="center">
+![Kirchhoff's First Law](images/kirchhoff_1.png)
+</div>
+
+$$
+I_1 + I_2 = I_3 + I_4 + I_5 \\
+I_1 + I_2 - I_3 - I_4 - I_5 = 0
+$$
+
+### Kirchhoff's Second Law
+
+Around any closed loop in a circuit, the directed sum of potential differences across components is zero.
+
+$$
+\sum_{k=1}^{n} U_k = 0
+$$
+
+<div align="center">
+![Kirchhoff's Second Law](images/kirchhoff_2.png)
+</div>
+
+$$
+-U_A - U_B - U_C + U_D + U_E = 0
+$$
+
+### Voltage divider
+
+There are many types of voltage dividers, being named according to the type of component elements: resistive divider, capacitive divider, compensated divider, etc...   
+The **resistive voltage divider** is obtained by applying a voltage `E` to a group of resistors in series and thus a fraction of the voltage applied to one of the resistors of the group can be obtained.
+
+<div align="center">
+![Voltage divider on GND](images/voltage_divider_gnd.png)
+</div>
+
+$$
+V_{out} = V_{in} * \frac{R_{2}}{R_{1} + R_{2}};
+$$
+
+:::warning
+Voltage divider can be considered a power source only if it operates in the empty state. It is **not recommended** to use a voltage divider to supply a circuit. It can be used to provide reference points.
+:::
+
+#### What happens if we use the voltage divider to supply a circuit.
+
+We consider a voltage divider that provides `3V3` from a `5V` power source `E` and a load resistance, `Rs`, representing the current consumption of a sensor or a circuit that needs to be supplied at `3V3`.
+
+<div align="center">
+![Voltage divider with load](images/voltage_divider_rs.png)
+</div>
+
+From Kirchhoff's First Law, the current through `R1` must be equal to the sum of the current through `R2` and `Rs`. 
+
+$$
+I_{R_1} = I_{R_2} + I_{R_S} 
+
+U = U_{E} * \frac{R_{2} || R_{S}}{R_{1} + R_{2} || R_{S}};
+$$
+
+The output voltage depends on the current intensity through `Rs`, on the current consumption of the circuit that needs to be supplied at `3V3`. This is not a viable power supply solution.
+
+:::note
+Beside the instability of the voltage divider with a load, the power rating for the `R1` must be suitable.
+
+$$
+P_{R_1} = U_{R_1} * I_{R_1}
+$$
+
+The power dissipation on the resistor is directly proportional with the current through the resistor. Resistor are fabricated with predefined power ratings, most common `1/4W`, `1/2W`, `1W`. 
```

Comment:
```suggestion
The power dissipation on the resistor is directly proportional with the current through the resistor. Resistor are fabricated with predefined power ratings, most common $$\frac{1}{4}W$$, `1/2W`, `1W`. 
```

Not sure about this though.

Reply:
Not sure about the statement, or about the style?